src/mmcmds.c

```diff
@@ -4244,56 +4240,56 @@ void writeExtractedSource(
         fixUndefinedLabels(extractNeeded, &tinyHdrComment);
         fprintf(fp, "%s", cat(tinyHdr, tinyHdrComment, hdrSuffix, NULL));
       }
-    } /* if header(s) needed */
+    } // if header(s) needed
 
-    /* Output $t statement if needed */
+    // Output $t statement if needed
     if (dollarTStmt == stmt) {
       fprintf(fp, "\n%s", dollarTCmt);
     }
 
-    /* Output statement if needed */
+    // Output statement if needed
     if (extractNeeded[stmt] == 'Y') {
       free_vstring(buf);
       buf = getDescriptionAndLabel(stmt);
 
       fixUndefinedLabels(extractNeeded, &buf);
 
       fprintf(fp, "%s", buf);
-      if (stmt == g_statements + 1) bug(272); /* Text below last statement
-           isn't (currently) used - do we need it? */
+      // Text below last statement isn't (currently) used - do we need it?
+      if (stmt == g_statements + 1) bug(272); 
       if (stmt != g_statements + 1) {
-        extractedStmts++; /* For final message */
+        extractedStmts++; // For final message
         fprintf(fp, "$%c", g_Statement[stmt].type);
         if (g_Statement[stmt].type != lb_ && g_Statement[stmt].type != rb_) {
-          /* $v $c $d $e $f $a $p */
+          // $v $c $d $e $f $a $p
           let(&buf, space(g_Statement[stmt].mathSectionLen));
           memcpy(buf, g_Statement[stmt].mathSectionPtr,
               (size_t)(g_Statement[stmt].mathSectionLen));
           fprintf(fp, "%s", buf);
           if (g_Statement[stmt].type != p_) {
             fprintf(fp, "$.");
-/*D*//*fprintf(fp, "#%ld#",stmt);*/
+// D // fprintf(fp, "#%ld#",stmt);
```

Comment:
```suggestion
/*D*/ // fprintf(fp, "#%ld#",stmt);
```
I forget what style exactly you were using for these but you should keep it consistent

Reply:
Ah yes, I left all `/*E*/` comments in the original form, I must have missed that this `/*D*/` comment was of the same kind.